matchbox-server/with-ca/application.yaml

```diff
@@ -11,12 +11,12 @@ hapi:
         url: classpath:/hl7.fhir.r4.core.tgz
       fhir_terminology:
```

Comment:
We should check if we could define all those objects in the main yaml file, to allow injection in all other configuration files. That would be very neat, but I'm unsure if Spring Boot supports it.

Reply:
we can't, that's why we had to take them out, it does ntot add to arrays
and for r5 we need a different set of tpyes

Reply:
that would be nice indeed, let's try that

Reply:
and extensions needs to be before terminology for matchbox server (circular dependency)